libs/plugin-analysis-api/build.gradle

```diff
@@ -9,6 +9,7 @@
 apply plugin: 'elasticsearch.publish'
 apply plugin: 'elasticsearch.build'
 
+group = "org.elasticsearch.plugin.analysis"
```

Comment:
I think it would cleaner to have this directly under `org.elasticsearch.plugin`. There is no need for a unique group per artifact.